drivers/aks/aks_driver.go

```diff
@@ -27,6 +28,9 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+var truePointer = true
+var redactionRegex = regexp.MustCompile("\"(clientId|secret)\": \"(.*)\"")
```

Comment:
can these be made into `const`s?

Reply:
Nein. 
- `truePointer` is used for a pointer assignment, and you can't take the address of a const.
- The regex is returned from a function and can't be made into a const

Reply:
shows what I know, very well